twisted/plugins/tracker_plugin.py

```diff
@@ -50,6 +52,10 @@
 else:
     SOCKET_BLOCK_ERRORCODE = errno.EWOULDBLOCK
 
+# Remove the limited history observer log
```

Comment:
maybe move this to the other module and add a link to the issue so we can track its evolution?